doc/classes/Translation.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Translation" inherits="Resource" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Language Translation.
+		A language translation that maps a string to another string.
```

Comment:
This sounds as if a translation handled only a single string...

Reply:
I didn't test out this class thoroughly - I just took this from the full description. What would be accurate here?

Reply:
Well a Translation of is a collection of string-string mappings. It's correct that it maps a string to another string, but there are many strings in a Translation. It's kind of like a dictionary.

Reply:
Okay, I fixed it in both places.

> A language translation that maps a collection of strings to their individual translations.